inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -24,95 +22,127 @@
 from inductiva.utils import files
 
 
-class WindTunnelSimulationOutput:
+class WindTunnelOutput:
     """Post-Process WindTunnel simulation outputs.
 
     Current Support:
         OpenFOAM
     """
 
-    def __init__(self, sim_output_path: Path, time_step: int = 100):
+    def __init__(self, sim_output_path: Path):
         """Initializes a `WindTunnelSimulationOutput` object.
 
         Args:
             sim_output_path: Path to simulation output files.
-            time_step: Time step where we read the data.
         """
 
         self.sim_output_path = sim_output_path
-        self.time_step = time_step
-        self.object_data = self.get_object_data()
 
-    def get_object_data(self):  # pylint: disable=unused-argument
+    def get_mesh_data(self, time_step):  # pylint: disable=unused-argument
         """Get aerodynamics data over an object inside the WindTunnel.
 
         Current Support - OpenFOAM
         """
 
-        reading_file = os.path.join(self.sim_output_path, "foam.foam")
+        # The OpenFOAM data reader from PyVista requires that a file named
+        # "foam.foam" exists in the simulation output directory.
+        # Create this file if it does not exist.
+        foam_file_path = os.path.join(self.sim_output_path, "foam.foam")
+        pathlib.Path(foam_file_path).touch(exist_ok=True)
 
-        # Create reading file
-        with open(reading_file, "w", encoding="utf-8") as file:
-            file.close()
+        reader = pv.OpenFOAMReader(foam_file_path)
+        reading_file = os.path.join(self.sim_output_path, "foam.foam")
 
         # Initialize reader and define reading time-step
         reader = pv.POpenFOAMReader(reading_file)
-        reader.set_active_time_value(self.time_step)
+        reader.set_active_time_value(time_step)
 
-        mesh = reader.read()
-        object_data = mesh["boundary"]["object"]
+        full_mesh = reader.read()
+        domain_mesh = full_mesh["internalMesh"]
+        object_mesh = full_mesh["boundary"]["object"]
 
-        return object_data
+        return domain_mesh, object_mesh
 
-    def get_physical_field(self, physical_property: str = "pressure"):
+    def get_physical_field(self,
+                           physical_property: str = "pressure",
+                           time_step: float = 50):
```

Comment:
Is this a time? Or the index of a time step? Make this explicit in the docstring.